### PR TITLE
[BugFix] alter judgetask to do a full judging

### PR DESCRIPTION
**Pull request by mvr320**
closes https://github.com/DOMjudge/domjudge/issues/1096

### CODE REVIEW

**Comment by meisterT** [comment]
I am not sure the fix is what we want.

The column hostname is gone since 0036cdaf8454b164ea9f677dd217a1000de2d072 (cc @nickygerritsen) but it didn't get deleted but was replaced with judgehost. I /assume/ but didn't double check that we probably want to change the condition to check for hostname. I'll look more closely later today.

**Comment by mvr320** [comment]
> 
> 
> I am not sure the fix is what we want.
> 
> The column hostname is gone since [0036cda](https://github.com/DOMjudge/domjudge/commit/0036cdaf8454b164ea9f677dd217a1000de2d072) (cc @nickygerritsen) but it didn't get deleted but was replaced with judgehost. I /assume/ but didn't double check that we probably want to change the condition to check for hostname. I'll look more closely later today.

Ok, I wrongly assumed that this was part of the API changes, I'll first make some tests for this and see if I can alter the code to set the hostname.

File: webapp/tests/Unit/Controller/Jury/SubmissionControllerTest.php [hunk 1]
```diff
@@ -38,4 +38,20 @@ public function provideViews(): Generator
             }
         }
     }
+
+    /*
+     * When lazy evaluation is enabled testcases are evaluated until a non-correct
```
Comment (meisterT) [inline webapp/tests/Unit/Controller/Jury/SubmissionControllerTest.php:43]: Remove this? Or fix?

**Comment by mvr320** [comment]
As this is now purely about the tests and I still have a lot of those small branches with tests I'll do this in a different PR.

**Pull request closed without merging**